catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/converter/CswRecordConverter.java

```diff
@@ -224,6 +230,32 @@ public Metacard transform(InputStream inputStream, String id)
     public BinaryContent transform(Metacard metacard, Map<String, Serializable> arguments)
             throws CatalogTransformerException {
         StringWriter stringWriter = new StringWriter();
+        if (StringUtils.isNotBlank(metacard.getMetadata())) {
```

Comment:
you can assign the getMetadata call to a string and not need the subsequent calls. not sure how much more performant or legible it would be though